fluentinterface/src/main/java/com/iluwatar/fluentinterface/fluentiterable/FluentIterable.java

```diff
@@ -84,25 +84,25 @@
    * @param <NEW_TYPE> the target type of the transformation
    * @return a new FluentIterable of the new type
    */
-  <NEW_TYPE> FluentIterable<NEW_TYPE> map(Function<? super TYPE, NEW_TYPE> function);
+  <NEW_TYPE> FluentIterable<NEW_TYPE> map(Function<? super T, NEW_TYPE> function);
```

Comment:
We can also change `NEW_TYPE` to be `E` or some other token and javadoc what that token represents.
Suggestion: Change the default generic token from `T` to `E` and change `NEW_TYPE` to `T`.